src/is.test.ts

```diff
@@ -33,16 +33,319 @@ import {
+  });
+});
+
+// isIntegerArray(value)
+describe('isIntegerArray(value)', () => {
+  it('Should be a function', () => {
+    expect(isIntegerArray).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type number[] and each array entry is an integer', () => {
+    expect(isIntegerArray([0, 0, 0])).to.equal(true);
+    expect(isIntegerArray([1, 2, 3])).to.equal(true);
+    expect(isIntegerArray([0.00, 1.00, 2.00, 3.00])).to.equal(true);
+    expect(isIntegerArray([])).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type number[]', () => {
+    expect(isIntegerArray('This is a String')).to.equal(false);
+    expect(isIntegerArray(100)).to.equal(false);
+    expect(isIntegerArray(undefined)).to.equal(false);
+    expect(isIntegerArray(null)).to.equal(false);
+    expect(isIntegerArray(true)).to.equal(false);
+    expect(isIntegerArray(false)).to.equal(false);
+    expect(isIntegerArray(['String', 2, false])).to.equal(false);
+    expect(isIntegerArray({})).to.equal(false);
+    expect(isIntegerArray()).to.equal(false);
+  });
+
+  it('Should return false when "value" is type number[] but not every array entry is an integer', () => {
+    expect(isIntegerArray([1.1, 2.2, 3.3])).to.equal(false);
+    expect(isIntegerArray([1, 2.2, 3.3])).to.equal(false);
+    expect(isIntegerArray([1, 2, 3.3])).to.equal(false);
+  });
+});
+
+// isNaN(value)
+describe('isNaN(value)', () => {
+  it('Should be a function', () => {
+    expect(isNaN).to.be.a('function');
+  });
+
+  it('Should return true when "value" is NaN', () => {
+    expect(isNaN(NaN)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not NaN', () => {
+    expect(isNaN('This is a String')).to.equal(false);
+    expect(isNaN(100)).to.equal(false);
+    expect(isNaN(undefined)).to.equal(false);
+    expect(isNaN(null)).to.equal(false);
+    expect(isNaN(true)).to.equal(false);
+    expect(isNaN(false)).to.equal(false);
+    expect(isNaN([])).to.equal(false);
+    expect(isNaN({})).to.equal(false);
+    expect(isNaN()).to.equal(false);
+  });
+});
+
+// isNull(value)
+describe('isNull(value)', () => {
+  it('Should be a function', () => {
+    expect(isNull).to.be.a('function');
+  });
+
+  it('Should return true when "value" is null', () => {
+    expect(isNull(null)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not null', () => {
+    expect(isNull('This is a String')).to.equal(false);
+    expect(isNull(100)).to.equal(false);
+    expect(isNull(undefined)).to.equal(false);
+    expect(isNull(true)).to.equal(false);
+    expect(isNull(false)).to.equal(false);
+    expect(isNull([])).to.equal(false);
+    expect(isNull({})).to.equal(false);
+    expect(isNull()).to.equal(false);
+  });
+});
+
+// isObject(value)
+describe('isObject(value)', () => {
+  it('Should be a function', () => {
+    expect(isObject).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type any{}', () => {
```

Comment:
any{} is not a valid type:

```suggestion
  it('Should return true when "value" is type object', () => {
```